apps/main/src/lend/store/createOhlcChartSlice.ts

```diff
@@ -308,35 +309,40 @@ const createOhlcChart = (set: SetState<State>, get: GetState<State>) => ({
       const network = networks[chainId].id.toLowerCase()
 
       try {
-        const llammaOhlcFetch = await fetch(
-          `https://prices.curve.fi/v1/lending/llamma_ohlc/${network}/${poolAddress}?agg_number=${interval}&agg_units=${timeUnit}&start=${start}&end=${end}`,
-        )
-        const llammaOhlcResponse: LlammaOhlcApiResponse = await llammaOhlcFetch.json()
+        const ohlc = await getOHLC({
+          endpoint: 'lending',
+          chain: network as Chain,
+          llamma: poolAddress as Address,
+          interval,
+          units: timeUnit as Parameters<typeof getOHLC>[0]['units'],
```

Comment:
Should we introduce a `TimeUnit` type?

Reply:
The API is rather inconsistent with its time options, hence I 'hardcoded' them instead of defining a type.